src/Payum/Core/Tests/Bridge/Doctrine/Storage/DoctrineStorageTest.php

```diff
@@ -134,18 +134,18 @@ public function shouldFindModelById()
     }
 
     /**
-     * @return \PHPUnit_Framework_MockObject_MockObject|\Doctrine\Common\Persistence\ObjectManager
+     * @return \PHPUnit_Framework_MockObject_MockObject|\Doctrine\Persistence\ObjectManager
      */
     protected function createObjectManagerMock()
     {
-        return $this->createMock('Doctrine\Common\Persistence\ObjectManager');
+        return $this->createMock('Doctrine\Persistence\ObjectManager');
```

Comment:
As a follow up to my previous comment. If you agree with me on using `::class` i would consider changing this also, it is already a change introduced and will help to keep consistency throughout the code.